src/Resources/config/core.php

```diff
@@ -151,6 +152,11 @@
 
         ->alias(SearchHandler::class, 'sonata.admin.search.handler')
 
+        ->set('sonata.admin.controller.crud', CRUDController::class)
+            ->public()
+            ->autowire()
```

Comment:
If I don't use autowire, this does not get the auto injected container.

Reply:
I've injected a `Psr\Container\ContainerInterface`, I think it works fine.

https://github.com/sonata-project/SonataAdminBundle/pull/6476/commits/2b0a304a2daffc3132396d5988a28865d482d108